internal/publish/database/exposure.go

```diff
@@ -368,7 +407,7 @@ func executeReviseExposure(ctx context.Context, tx pgx.Tx, stmtName string, exp
 }
 
 // InsertAndReviseExposures transactionally revises and inserts a set of keys as necessary.
-func (db *PublishDB) InsertAndReviseExposures(ctx context.Context, incoming []*model.Exposure, token *pb.RevisionTokenData, tokenRequired bool) (int, error) {
+func (db *PublishDB) InsertAndReviseExposures(ctx context.Context, incoming []*model.Exposure, token *pb.RevisionTokenData, tokenRequired, bulkInsert bool) (int, error) {
```

Comment:
let's not put bulk insert in this same method.

bulk insert is meant for performance testing and shouldn't be in the main code path.

Reply:
Sounds good, I will break it out

Reply:
@mikehelmick did you want it out of this package entirely or just broken out of `InsertAndReviseExposures` and make a new `BulkInsertExposures`

Reply:
can be in this package - but should be in a separate file, clearly marked as test infrastructure